.env

```diff
@@ -2,4 +2,5 @@ CONTENTFUL_SPACE_ID=1kfhsqlc8ewi
 CONTENTFUL_API_KEY=wTGnaFHcz9McLX2KPwp-6cArrlYRraKrtvhVI3F-_F0
 NEXT_PUBLIC_CONTENTFUL_SPACE_ID=1kfhsqlc8ewi
 NEXT_PUBLIC_CONTENTFUL_API_KEY=wTGnaFHcz9McLX2KPwp-6cArrlYRraKrtvhVI3F-_F0
+NEXT_PUBLIC_CONTENTFUL_IS_PREVIEW=true
```

Comment:
this property depends on the enviroment that we are using, I think that it should not be uploaded